docs/eventing/getting-started.md

```diff
@@ -1,163 +1,88 @@
 
-   ```sh
-   kubectl --namespace event-example get Broker default
-   ```
+    ```
+    kubectl create -f - <<EOF
+    apiVersion: eventing.knative.dev/v1
+    kind: Broker
+    metadata:
+     name: default
+     namespace: event-example
+    EOF
+    ```
 
-   This shows the `Broker` that you created:
+1. Verify that the broker is working correctly, by entering the following command:
 
-   ```sh
-   NAME      READY   REASON   URL                                                        AGE
-   default   True             http://default-broker.event-example.svc.cluster.local      1m
-   ```
+    ```
+    kubectl --namespace event-example get Broker default
+    ```
 
-   When the `Broker` has the `READY=True` state, it can begin to manage any
-   events it receives.
+    This shows information about your broker. If the broker is working correctly, it shows a `READY` status of `True`:
 
-1. If `READY=False`, wait 2 minutes and re-run the command. If you continue to
-   receive the `READY=False`, see the [Debugging Guide](./debugging/README.md)
-   to help troubleshoot the issue.
+    ```
+    NAME      READY   REASON   URL                                                        AGE
+    default   True             http://default-broker.event-example.svc.cluster.local      1m
+    ```
 
-Now that your `Broker` is ready to manage events, you can create and configure
-your event producers and consumers.
+    If `READY` is `False`, wait a few moments and then run the command again.
+    If you continue to receive the `False` status, see the [Debugging Guide](./debugging/README.md) to troubleshoot the issue.
 
-### Creating event consumers
+## Creating event consumers
 
-Your event consumers receive the events sent by event producers. In this step,
-you will create two event consumers, `hello-display` and `goodbye-display`, to
-demonstrate how you can configure your event producers to selectively target a
-specific consumer.
+In this step, you create two event consumers, `hello-display` and `goodbye-display`, to
+demonstrate how you can configure your event producers to target a specific consumer.
 
 1. To deploy the `hello-display` consumer to your cluster, run the following
    command:
 
-   ```sh
-   kubectl --namespace event-example apply --filename - << END
-   apiVersion: apps/v1
-   kind: Deployment
-   metadata:
-     name: hello-display
-   spec:
-     replicas: 1
-     selector:
-       matchLabels: &labels
-         app: hello-display
-     template:
-       metadata:
-         labels: *labels
-       spec:
-         containers:
-           - name: event-display
-             # Source code: https://github.com/knative/eventing-contrib/tree/master/cmd/event_display
-             image: gcr.io/knative-releases/knative.dev/eventing-contrib/cmd/event_display
-
-   ---
-
-   # Service pointing at the previous Deployment. This will be the target for event
-   # consumption.
+     ```
+     kubectl --namespace event-example apply --filename - << END
```

Comment:
L32 (broker creation uses (and I prefer)):
kubectl --namespace event-example apply -f - << EOF

So, using the EOF as the limiter (for consistency)
using -f instead of --filename (shorter to type, and for consistency).

Reply:
would it makes sense to document writing style conventions somewhere?

Reply:
Yep, we're planning to do this in the docs WG just haven't had bandwidth thus far.